circuit/src/get_account.nr

```diff
@@ -1,14 +1,25 @@
+use dep::proof;
+
 struct Account {
     balance: Field,
     codeHash: Field,
     nonce: Field,
+    state_root: [u8; 32],
+    key: [u8; 20],
+    value: [u8; 134],
+    proof: [u8; 4256],
+    depth: Field,
```

Comment:
- I would call it `AccountWithProof`
- I wonder why value is 134? Is it combination of 4 fields (balance, nonce, contracthash, storageRoot)?
Could be done, with other changes in a separate ticket/PR. #nextPR

Reply:
Changed `Account` to `AccountWithProof`; Replaced `134` value with import `proof::MAX_ACCOUNT_STATE_LENGTH`